src/spider/scheduler/scheduler.cpp

```diff
@@ -251,5 +268,11 @@ auto main(int argc, char** argv) -> int {
         spdlog::error("Failed to join thread: {}", e.what());
     }
 
+    // If SIGTERM was caught and StopFlag is requested, set the exit value to corresponding to
+    // SIGTERM.
```

Comment:
```suggestion
    // If SIGTERM was caught and StopFlag is requested, set the exit value corresponding to SIGTERM.
```
Sorry this might have been my typo from a previous suggestion.